integration-tests/go-sql-server-driver/concurrent_gc_test.go

```diff
@@ -136,21 +147,39 @@ func (gct gcTest) doUpdate(t *testing.T, ctx context.Context, db *sql.DB, i int)
 		}
 		t.Logf("err in Exec update: %v", err)
 	}
-	if gct.commit {
-		_, err = conn.ExecContext(ctx, fmt.Sprintf("call dolt_commit('-am', 'increment vals id = %d')", i))
-		if gct.sessionAware {
-			assert.NoError(t, err)
-		} else if err != nil {
-			if !assert.NotContains(t, err.Error(), "dangling ref") {
-				return err
-			}
-			if !assert.NotContains(t, err.Error(), "is unexpected noms value") {
-				return err
+	if err == nil {
```

Comment:
Probably simpler to early return if err != nil here (also add a note why that's fine in some cases)